requirements.txt

```diff
@@ -1,3 +1,19 @@
+aenum==2.2.3
+astroid==2.3.3
 beautifulsoup4==4.8.2
+certifi==2019.11.28
+chardet==3.0.4
+colorama==0.4.3
+config==0.4.2
+idna==2.9
+isort==4.3.21
+lazy-object-proxy==1.4.3
+mccabe==0.6.1
+pylint==2.4.4
+requests==2.23.0
+six==1.14.0
 soupsieve==1.9.5
+typed-ast==1.4.1
+urllib3==1.25.8
```

Comment:
requirements.txt is already changed as a part of this [PR](https://github.com/DeploySolutionsASU/SER517-ClimateChangeSpider/pull/25). Can you revert the changes made here? @Narendramohanm